### ISSUE TITLE
openshift_master | fail

### ISSUE THREAD

**Issue by johnjelinek** [openshift/openshift-ansible#368]
```
TASK: [openshift_master | fail ] **********************************************
failed: [osv3-master.johnjelinek.com] => {"failed": true}
msg: openshift_master_cluster_password must be set for multi-master installations
```

I only have one master. I don't think I should need a cluster password.

**Comment by goern** [comment]
this seems to be fixed in master by #374 close?

**Comment by detiber** [comment]
@tweist please close
